README.md

```diff
@@ -12,20 +12,21 @@ It's still very much a work in progress.
 
 ## Setting Up The Dev Environment
 
-You must have the following packages installed globally on your system:
+You must have the following tools installed on your system (or in a docker container):
 
 * NodeJS 14.6.0
 * Yarn 1.22.4
-* TypeScript 3.9.7
-* Angular 10.0.4
-* NestJS 7.4.1
 * MongoDB 4.2.1
-* [Neon](https://neon-bindings.com/docs/getting-started) and the [Rust 1.15 (or later)toolchain](https://rustup.rs/).
-    * (Optionally) If not on Ubuntu 16.10 or later, you will also need [LLVM](https://releases.llvm.org/download.html), as native module compilation relies on LLVM's `libclang`. You will also need to set the `LIBCLANG_PATH` environment variable to LLVM's `/bin` directory.
+* The [Rust 1.15 (or later) toolchain](https://rustup.rs/).
+    * You may also need the following packages in order to compile the Rust packages:
+        * `build-essential`
+        * `llvm-dev`
+        * `libclang-dev`
+        * `clang`
```

Comment:
Note: We may not actually need these anymore! I need to rebuild the docker container without them and see if the Rust modules still build.